### ISSUE TITLE
Default parameter not valid ECMAScript prior to version ES6

### ISSUE THREAD

**Issue by mwittig** [coolchip/luxtronik2#1]
The default parameter used in the [function declaration luxtronik](https://github.com/coolchip/luxtronik2/blob/master/luxtronik.js#L19) is not valid ECMAScript prior to version ES6. As coffee-script currently only supports ES5 your library cannot be used as part of coffee-script projects.  It would nice if this could be changed by removing the default parameter value from the declaration and adding some code instead like the following:

```
if (typeof this._port === 'undefined')  {
        this._port = 8888;
}
```

**Comment by coolchip** [comment]
Fixed in v 0.0.2 ;)

```javascript
function luxtronik(host, port) {
    if (typeof port === "undefined") {
        this._port = 8888;
    } else {
        this._port = port;
    }
    this._host = host;
}
```

Thank you very much your help.

**Comment by mwittig** [comment]
Thanks for covering this so quickly. I have found some more issues and filed a pull request for that.
